
From: <mike.miller@hp.com>

This patch changes our read_ahead to 1024.  This has been shown to increase
performance.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/block/cciss.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/block/cciss.c~cciss-update-7-read_ahead-bumped-to-1024 drivers/block/cciss.c
--- 25/drivers/block/cciss.c~cciss-update-7-read_ahead-bumped-to-1024	Fri Aug  6 15:01:52 2004
+++ 25-akpm/drivers/block/cciss.c	Fri Aug  6 15:01:52 2004
@@ -115,7 +115,7 @@ static struct board_type products[] = {
 /*define how many times we will try a command because of bus resets */
 #define MAX_CMD_RETRIES 3
 
-#define READ_AHEAD 	 256
+#define READ_AHEAD 	 1024
 #define NR_CMDS		 384 /* #commands that can be outstanding */
 #define MAX_CTLR 8
 
_
